### PR TITLE
databases-week1/andrii

### PR DESCRIPTION
**Pull request by ayaroshenkocom**

### CODE REVIEW

File: 5_databases/week1/homework.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+FROM task
+WHERE title LIKE '%database%' OR description LIKE '%database%';
+
+--> 8. Get the title and status (as text) of all tasks
+SELECT task.title, status.name AS status_text
+FROM task
+JOIN status on task.status_id = status_id
+
+--> 9. Get the name of each status, along with a count of how many tasks have that status
+SELECT status.name AS status_name, COUNT(*) AS task_count
```
Comment (adionditsak) [inline 5_databases/week1/homework.sql:43]: This is the same problem as above, it returns the tasks 3 times. The current output is:


```
+-------------+----------------+
| name        | COUNT(task.id) |
+-------------+----------------+
| Done        |             35 |
| In progress |             35 |
| Not started |             35 |
+-------------+----------------+
```
If you fix the SQL statement above this should be fixed as well.

File: 5_databases/week1/homework.sql [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+FROM task
+JOIN status ON task.status_id = status.id
+GROUP BY status.name;
+
+--> 10. Get the names of all statuses, sorted by the status with most tasks first
+SELECT status.name AS status_name, COUNT(*) AS task_count
```
Comment (adionditsak) [inline 5_databases/week1/homework.sql:49]: Same as above.

Good job!

**Review by adionditsak — CHANGES_REQUESTED** [review]

File: 5_databases/week1/homework.sql [hunk 3]
```diff
@@ -0,0 +1,53 @@
+--> 1. Find out how many tasks are in the task table
+SELECT COUNT(*) AS hom_many_tasks
```
[the remark this answers is not in the document]
Comment (ayaroshenkocom) [reply 5_databases/week1/homework.sql:2]: Ok. Thank you

File: 5_databases/week1/homework.sql [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+FROM task;
+
+--> 2. Find out how many tasks in the task table do not have a valid due date
+SELECT  COUNT(*) AS task_no_due_data
+FROM task
+WHERE due_date IS NULL
+
+--> 3. Find all the tasks that are marked as done
+SELECT task.*
```
[the remark this answers is not in the document]
Comment (ayaroshenkocom) [reply 5_databases/week1/homework.sql:11]: In fact, on my first try I thought of doing just that. But then I tried to find another way :)

File: 5_databases/week1/homework.sql (hunk 1 above)
Comment (ayaroshenkocom) [reply 5_databases/week1/homework.sql:43]: Are your sure? Because I have this output https://skr.sh/sMrEcTGaNAp

Comment (ayaroshenkocom) [reply 5_databases/week1/homework.sql]: But I made the changes anyway :)

File: 5_databases/week1/homework.sql (hunk 2 above)
Comment (ayaroshenkocom) [reply 5_databases/week1/homework.sql]: Thank you for the review :)

File: 5_databases/week1/homework.sql (hunk 1 above)
Comment (adionditsak) [reply 5_databases/week1/homework.sql]: @ayaroshenkocom Ah, this was actually correct, sorry. Good job. The task above this one has to be fixed tho :)

Comment (ayaroshenkocom) [reply 5_databases/week1/homework.sql]: Ok. Thank you very much for the review :)

**Comment by ayaroshenkocom** [comment]
Changes have been made. Thanks for the review.

**Pull request closed without merging**